apps/new-dealer/index.js

```diff
@@ -20,17 +19,11 @@ const submission = Submission({
 });
 
 const pdf = require('../common/behaviours/pdf-upload');
-
-const Emailer = require('../common/behaviours/emailer');
-const emailer = Emailer({
-  template: path.resolve(__dirname, './emails/confirm.html'),
+const templateId = config.govukNotify.templateSection5;
+const getContactSendEmail = require('./behaviours/get-contact-send-email')({
```

Comment:
This is the only time that the common SendEmail behaviour is not being reused because we have to do some logic to work out the name of the contact person
https://github.com/UKHomeOffice/firearms/pull/407/files#diff-c5019d40e4f3314602f7c9cb826699dda8f2bfa8365a6437a536594701b07902